src/psyclone/psyir/transformations/parallel_loop_trans.py

```diff
@@ -160,6 +298,8 @@ def validate(self, node, options=None, **kwargs):
             execution.
         :param bool privatise_arrays: whether to declare as private any
             write after write dependency symbols.
+        :param bool enable_reductions: whether to attempt parallelisation
+            of loops containing reductions.
```

Comment:
All the :param: options on validate apart from `node` and `options` should be removed as part of this, since this uses the `transformation_docstring_wrapper`.